README-dev.md

```diff
@@ -1,98 +1,136 @@
 # Mina
 
-Mina is a new cryptocurrency protocol with a lightweight, constant-sized blockchain.
+Mina is a cryptocurrency protocol with a lightweight, constant-sized blockchain.
 
-- [Node Developers Overview Page](https://docs.minaprotocol.com/node-developers)
-- [Repository Readme](README.md)
+- [Node Developers Overview](https://docs.minaprotocol.com/node-developers)
+- [Mina README](README)
 
-If you haven't seen it yet, [CONTRIBUTING.md](CONTRIBUTING.md) has information
-about our development process and how to contribute. If you just want to build
+For information about our development process and how to contribute, see [CONTRIBUTING.md](CONTRIBUTING). If you want to build
 Mina, this is the right file!
 
 ## Building Mina
 
-Building Mina can be slightly involved. There are many C library dependencies that need
-to be present in the system, as well as some OCaml-specific setup.
+Building Mina is involved because many C library dependencies must be present in the system. OCaml-specific setup is also required.
 
-If you are already a Nix user, or are comfortable installing Nix, it
-can be an easy way to build Mina locally. See
-[nix/README.md](./nix/README.md) for more information and
-instructions.
+If you are already a Nix user, or are comfortable installing Nix, you already have a way to build Mina locally. For information and
+instructions, see [nix/README.md](./nix/README.md).
 
-Currently, Mina builds/runs on Linux & macOS. MacOS may have some issues that you can track [here](https://github.com/MinaProtocol/coda/issues/962).
+Mina builds and runs on Linux and macOS.
```

Comment:
We can omit "currently"
https://developers.google.com/style/timeless-documentation